docs/doc1.md

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
Obviously further work will be required to get this done.
I would recommend you to clean not needed files (like this one and then we can push it to upstream branch so many people can collaborate.